src/map/npc.h

```diff
@@ -86,6 +86,7 @@ struct npc_data {
 	unsigned size : 2;
 
 	struct status_data status;
+	struct status_change sc;
```

Comment:
look like this related to npc statuses but not clan system?
may be better move npc statuses into separate commit?